modules/codegen/src/main/scala/com/twilio/guardrail/generators/AkkaHttpServerGenerator.scala

```diff
@@ -305,7 +305,7 @@ object AkkaHttpServerGenerator {
       directivesFromParams(
         arg => tpe => Target.pure(q"formField(Symbol(${arg}).as[${tpe}])"),
         arg => tpe => Target.pure(q"formField(Symbol(${arg}).as[${tpe}].*)"),
-        arg => tpe => Target.pure(q"formField(Symbol(${arg}).as[${tpe}].*).map(xs => Option(xs).filterNot(_.isEmpty)).apply"),
+        arg => tpe => Target.pure(q"formField(Symbol(${arg}).as[${tpe}].*).map(xs => Option(xs).filterNot(_.isEmpty))"),
```

Comment:
I had to update akkaHttp for `multiOpt` version only. I hope this is correct. Have no experience with akkaHttp.